Sources/Plugins/ExternalSensorsPlugin/BLE/BLEPairedSensors/BLEPairedSensorsViewController.swift

```diff
@@ -171,7 +171,7 @@ final class BLEPairedSensorsViewController: OABaseNavbarViewController {
             }
             onSelectDeviceAction?(currentSelectedDevice)
         }
-        tableView.reloadData()
```

Comment:
I think tableView.reloadData() should be kept. The user wants to see what they selected before the controller closes.